commit 4a976200d7853588336005a394dd31d905f5caf6
Author: Philip Langdale <philipl@overt.org>
Date:   Wed Oct 31 16:58:09 2018 -0700

    avcodec/vdpau: Initialise driver version variable explicitly
    
    If the identification string ever changed, we might not match anything
    in the sscanf(). At least have predictable behaviour.

diff --git a/libavcodec/vdpau.c b/libavcodec/vdpau.c
index 59454e8c4b..167f06d7ae 100644
--- a/libavcodec/vdpau.c
+++ b/libavcodec/vdpau.c
@@ -208,7 +208,7 @@ int ff_vdpau_common_init(AVCodecContext *avctx, VdpDecoderProfile profile,
         return vdpau_error(status);
     if (avctx->codec_id == AV_CODEC_ID_HEVC && strncmp(info_string, "NVIDIA ", 7) == 0 &&
         !(avctx->hwaccel_flags & AV_HWACCEL_FLAG_ALLOW_PROFILE_MISMATCH)) {
-        int driver_version;
+        int driver_version = 0;
         sscanf(info_string, "NVIDIA VDPAU Driver Shared Library  %d", &driver_version);
         if (driver_version < 410) {
             av_log(avctx, AV_LOG_VERBOSE, "HEVC with NVIDIA VDPAU drivers is buggy, skipping.\n");
